openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -317,7 +317,7 @@ def _get_upsell_information_for_schedule(user, schedule):
             enrollment.dynamic_upgrade_deadline,
             get_format(
                 'DATE_FORMAT',
-                lang=course.language,
+                lang=course.closest_released_language,
```

Comment:
Same here, just want to be sure that this will be ok if `closest_released_language` is `None`.